### PR TITLE
GET /v1/samples/upsert/bulk/{jobId}/status API changes

### PR DESCRIPTION
**Pull request by shriramshankar**

### CODE REVIEW

File: api/v1/controllers/samples.js [hunk 1]
```diff
@@ -87,6 +89,36 @@ module.exports = {
...
   /**
+   * GET /samples/upsert/bulk/{key}/status
+   *
+   * Retrieves the status of the bulk upsert job and sends it back in the
+   * response
+   * @param {IncomingMessage} req - The request object
+   * @param {ServerResponse} res - The response object
+   * @param {Function} next - The next middleware function in the stack
+   */
+  getSampleBukUpsertStatus(req, res, next) {
```
Comment (iamigo) [inline api/v1/controllers/samples.js]: Buk >> Bulk

File: api/v1/swagger.yaml [hunk 2]
```diff
@@ -3625,6 +3625,37 @@ paths:
...
+  /samples/upsert/bulk/{key}/status:
+    x-swagger-router-controller: samples
+    get:
+      security:
+        - jwt: []
+      summary: Retrieve the specified sample
+      tags: [ samples ]
+      description: >-
+        Retrieve the status of the job identified by id in the request.
+      operationId: getSampleBukUpsertStatus
```
Comment (iamigo) [inline api/v1/swagger.yaml]: Buk >> Bulk

File: worker/jobs/bulkUpsertSamplesJob.js [hunk 3]
```diff
@@ -53,7 +53,9 @@ module.exports = (job, done) => {
       for (let i = 0; i < results.length; i++) {
         if (results[i].isFailed) {
           errorCount++;
-          errors.push(results[i]);
+
+          // we just need "explanation" to be added to the errors
+          errors.push(results[i].explanation);
```
Comment (iamigo) [inline worker/jobs/bulkUpsertSamplesJob.js]: Great!

**Comment by shriramshankar** [comment]
@iamigo @annyhe PR updated.

**Pull request merged**